### ISSUE TITLE
Crash starting vscode

### ISSUE THREAD

**Issue by lfrodrigues** [palantir/python-language-server#186]
I followed your instructions to install the language server but I'm getting the error bellow.

I'm using python-language-server master. Tried vscode 1.17.2 and insiders build and both fail with the same error. 

```
$ npm run vscode -- $PWD/../

> vscode-client@0.0.1 vscode /private/tmp/python-language-server/vscode-client
> npm run vscode:prepublish && VSCODE=$(which code-insiders || which code || echo echo ERROR: neither the code nor code-insiders vscode executable is installed); USER=dummy-dont-share-vscode-instance $VSCODE --user-data-dir=$PWD/.vscode-dev/user-data --extensionHomePath=$PWD/.vscode-dev/extensions --extensionDevelopmentPath=$PWD $* "/tmp/python-language-server/vscode-client/../"


> vscode-client@0.0.1 vscode:prepublish /private/tmp/python-language-server/vscode-client
> tsc -p ./

node_modules/vscode-languageclient/lib/client.d.ts(1,706): error TS2305: Module ''vscode'' has no exported member 'WorkspaceFolder'.
node_modules/vscode-languageclient/lib/workspaceFolders.proposed.d.ts(1,10): error TS2305: Module ''vscode'' has no exported member 'WorkspaceFoldersChangeEvent'.
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! vscode-client@0.0.1 vscode:prepublish: `tsc -p ./`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the vscode-client@0.0.1 vscode:prepublish script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/lfr/.npm/_logs/2017-11-04T18_46_54_371Z-debug.log
sh: --user-data-dir=/tmp/python-language-server/vscode-client/.vscode-dev/user-data: No such file or directory
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! vscode-client@0.0.1 vscode: `npm run vscode:prepublish && VSCODE=$(which code-insiders || which code || echo echo ERROR: neither the code nor code-insiders vscode executable is installed); USER=dummy-dont-share-vscode-instance $VSCODE --user-data-dir=$PWD/.vscode-dev/user-data --extensionHomePath=$PWD/.vscode-dev/extensions --extensionDevelopmentPath=$PWD $* "/tmp/python-language-server/vscode-client/../"`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the vscode-client@0.0.1 vscode script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/lfr/.npm/_logs/2017-11-04T18_46_54_393Z-debug.log
```

**Comment by gatesn** [comment]
If you open the command pane in VSCode you should run “Install shell commands”.

**Comment by lfrodrigues** [comment]
I did:

```
$ which code
/usr/local/bin/code
```

**Comment by gatesn** [comment]
Ah, re-reading your error, I suspect the versions of vscode and the vscode-languageclient aren't compatible. If you look at package.json, the versions aren't pinned.

**Comment by gatesn** [comment]
Fixed by #237
